### DISCUSSION TITLE
How are Icon updates scheduled?

### DISCUSSION THREAD

**Discussion by Andre601** [badges/shields#5369] [Q&A] [answered]
I was curious to know, how adding/supporting new icons from Simpleicons.org are scheduled.
Is it checked every x time and if new/changed icons exist, update, or is it only updated once Simpleicons gathered a large amount of new/changed/deleted icons?

I ask, because they now released new icons from which I would like to use one. But unfortunately does shields.io not yet display it when provided through the `logo` querry.

**Comment by calebcartwright** [comment, marked as answer]
Shields supports SimpleIcons via the [simple-icons](https://www.npmjs.com/package/simple-icons) package. We primarily integrate dependency updates via Dependabot which runs on the first day of the month, and that includes the simple-icons package. There's almost always a deployment to the Shields.io production environment within a couple days after the monthly dependency bumps, though that's more a practice than any kind of guarantee. 

If Dependabot tells us there's a new version of simple-icons then we'll pull it in, regardless of how many new icons there may or may not be.

> I ask, because they now released new icons from which I would like to use one. But unfortunately does shields.io not yet display it when provided through the logo querry.

I've described the process for pulling in SimpleIcons changes above, and I don't imagine it'll be feasible to increase the frequency there. When users would like to get a sense of when Shields.io will support some new X icon from SimpleIcons that's not yet available on Shields.io, I'd suggest:

1. Determine which [released version of SimpleIcons](https://www.npmjs.com/package/simple-icons) contains icon X
2. Check the [version of SimpleIcons used by Shields within this repo](https://github.com/badges/shields/blob/master/package.json)

If those two are the same, then it's probably a matter of days til the updates are finished getting deployed to production. If they are not, then it will probably be a week or so and you can keep an eye out for the weekly Dependabot update PR like #5357

**Comment by hwittenborn** [comment]
I'm trying to add the Arch Linux logo to a badge, but it doesn't appear to be working.

Any advice?

**Reply by calebcartwright** [reply]
hi @hwittenborn please open a new discussion or issue to discuss and we'll be happy to help!

**Comment by aliston-gomes** [comment]
Pinia icon is missing
